app/controllers/admin/orders_controller.rb

```diff
@@ -77,6 +77,6 @@ def order_params
   end
 
   def search_params
-    params.fetch(:search_term, {}).permit(:order_id, :ship_email, :ship_phone)
+    params.fetch(:order_search_term, {order_id: nil, ship_email: nil, ship_phone: nil}).permit(:order_id, :ship_email, :ship_phone)
```

Comment:
when will you query http://localhost:3000/admin/orders/search without any params?